### PR TITLE
DIS-118 Add upvote_count column to Post table

### PR DESCRIPTION
**Pull request by jogura**
* Add column to both create db scripts
* Change SQL to use upvote_count instead of ratings

@armonr @jimmordino

### CODE REVIEW

File: service/discussion/sql/create_discussion_db.sql [hunk 1]
```diff
@@ -112,6 +112,7 @@ CREATE TABLE IF NOT EXISTS `post` (
   `deleted_position` INT,
   `revision_id` BIGINT,
   `rating` INT,
```
Comment (armonr) [inline service/discussion/sql/create_discussion_db.sql]: Are we keeping rating?

**Comment by armonr** [comment]
LGTM

File: service/discussion/sql/create_discussion_db.sql (hunk 1 above)
Comment (jogura) [inline service/discussion/sql/create_discussion_db.sql]: For now, so we don't break existing services deployed we are doing an additive change. Once we have services deployed that do not reference `ratings` then we can delete the column.

Comment (armonr) [inline service/discussion/sql/create_discussion_db.sql]: Sounds good!

**Pull request merged**